tip-wlan/charts/wlan-portal-service/templates/statefulset.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   labels:
     {{- include "common.labels" . | nindent 4 }}
 spec:
+  podManagementPolicy: Parallel
```

Comment:
Are you sure this is going to fix anything? From [the docs](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#parallel-pod-management) this doesn't have any influence on how pods are being updated.

Reply:
it does fix update stuck errors in my experience

Reply:
https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#forced-rollback